core/src/main/res/values/strings.xml

```diff
@@ -122,26 +122,26 @@
     <string name="mark_all_read_label">Mark all as played</string>
     <string name="mark_all_read_msg">Marked all Episodes as played</string>
     <string name="mark_all_read_confirmation_msg">Please confirm that you want to mark all episodes as being played.</string>
-    <string name="mark_all_read_feed_confirmation_msg">Please confirm that you want to mark all episodes in this feed as being played.</string>
+    <string name="mark_all_read_feed_confirmation_msg">Please confirm that you want to mark all episodes in this podcast as being played.</string>
     <string name="mark_all_seen_label">Mark all as seen</string>
     <string name="mark_all_seen_msg">Marked all Episodes as seen</string>
     <string name="mark_all_seen_confirmation_msg">Please confirm that you want to mark all episodes as seen.</string>
     <string name="show_info_label">Show information</string>
-    <string name="show_feed_settings_label">Show feed settings</string>
-    <string name="feed_info_label">Feed info</string>
-    <string name="feed_settings_label">Feed settings</string>
+    <string name="show_feed_settings_label">Show podcast settings</string>
+    <string name="feed_info_label">Podcast info</string>
+    <string name="feed_settings_label">Podcast settings</string>
     <string name="rename_feed_label">Rename Podcast</string>
     <string name="remove_feed_label">Remove Podcast</string>
     <string name="share_label">Share&#8230;</string>
-    <string name="share_link_label">Share Link</string>
+    <string name="share_link_label">Share Episode URL</string>
+    <string name="share_link_with_position_label">Share Episode URL with Position</string>
     <string name="share_file_label">Share File</string>
-    <string name="share_link_with_position_label">Share Link with Position</string>
     <string name="share_feed_url_label">Share Feed URL</string>
-    <string name="share_item_url_label">Share Episode File URL</string>
-    <string name="share_item_url_with_position_label">Share Episode File URL with Position</string>
-    <string name="feed_delete_confirmation_msg">Please confirm that you want to delete the feed \"%1$s\" and ALL episodes of this feed that you have downloaded.</string>
-    <string name="feed_remover_msg">Removing Feed</string>
-    <string name="load_complete_feed">Refresh complete Feed</string>
+    <string name="share_item_url_label">Share Media File URL</string>
+    <string name="share_item_url_with_position_label">Share Media File URL with Position</string>
+    <string name="feed_delete_confirmation_msg">Please confirm that you want to delete the Podcast \"%1$s\" and ALL episodes of this feed that you have downloaded.</string>
+    <string name="feed_remover_msg">Removing Podcast</string>
+    <string name="load_complete_feed">Refresh complete Podcast</string>
```

Comment:
I am not sure if podcast should really be capitalized here

Reply:
Probably not, no. Just followed existing practice, but adapted to lowercase.